run-ib-pr-matrix.sh

```diff
@@ -49,6 +49,7 @@ pushd "$WORKSPACE/matrix-results"
   case "${TEST_FLAVOR}" in
     gpu )        MATRIX_ARGS="-w gpu ${MATRIX_ARGS}" ;;
     high_stats ) CMD_OPTS="-n 500" ; MATRIX_ARGS="-i all ${MATRIX_ARGS}" ;;
+    maxmem_profile ) CMD_OPTS="--maxmem_profile" ;;
```

Comment:
@gartung , is `--maxmem_profile` supported by all cmssw release cycles? If not then I would suggest to run `runTheMatrix.py --help` to see if this opt is supported or not. I n case a cmssw release cycle does not support it then just mark the commit status successfull with message `Memory profile not run due to lack of support`